cmd/sealer/cmd/cluster/run.go

```diff
@@ -218,6 +222,38 @@ func runWithClusterfile(clusterFile string, runFlags *types.RunFlags) error {
 	return createNewCluster(imageEngine, cf, imageSpec, runFlags.Mode)
 }
 
+func getClusterInstallInfo(labels map[string]string) clusterruntime.InstallInfo {
+	cri := labels["cluster.alpha.sealer.io/container-runtime-type"]
+	if cri == "" {
+		cri = "docker"
+	}
+	clusterRuntimeType := labels["cluster.alpha.sealer.io/cluster-runtime-type"]
+	if clusterRuntimeType == "" {
+		clusterRuntimeType = "kubernetes"
+	}
+	logrus.Infof("The cri is %s, crt is %s\n", cri, clusterRuntimeType)
+	return clusterruntime.InstallInfo{
+		ContainerRuntimeType: cri,
+		ClusterRuntimeType:   clusterRuntimeType,
+	}
+}
+
+func getClusterConfPath(labels map[string]string) string {
+	clusterRuntimeType := labels["cluster.alpha.sealer.io/cluster-runtime-type"]
+	if clusterRuntimeType == "" {
+		clusterRuntimeType = "/etc/kubernetes"
+	}
+	switch clusterRuntimeType {
+	case runtime.K8s:
+		return kubernetes.AdminKubeConfPath
+	case runtime.K0s:
+		return k0s.DefaultAdminConfPath
+	//TODO support k3s
```

Comment:
update comment to "support more kxs runtime here"

we can aim higher, hahaha.

Reply:
Oh, I will add kxs support in our official website runtime introduction after this pr be merged.
But I also think your suggestions are going to be constructive!